### PR TITLE
CMR-9260 creating the ingest and search draft concept API documents.

### PR DESCRIPTION
**Pull request by eereiter**

### CODE REVIEW

File: schemas/resources/schemas/collection-draft/v1.0.0/search.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+    </results>
+```
+
+##### JSON
+
+The JSON response includes the following fields.
+
+* hits - How many total Order Options were found.
+* took - How long the search took in milliseconds
+* items - a list of the current page of Order Options with the following fields
```
Comment (eudoroolivares2016) [inline schemas/resources/schemas/collection-draft/v1.0.0/search.md:86]:
```suggestion
* items - a list of the current page of draft collections with the following fields
```

File: schemas/resources/schemas/collection-draft/v1.0.0/search.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+    }
+```
+
+#### <a name="sorting-collection-draft-results"></a> Sorting Collection Draft Results
+
+By default, Collection Draft results are sorted by name, then by provider-id.
+
+One or more sort keys can be specified using the sort_key[] parameter. The order used impacts searching. Fields can be prepended with a - to sort in descending order. Ascending order is the default but + (Note: + must be URL encoded as %2B) can be used to explicitly request ascending.
+
+##### Valid Order Option Sort Keys
```
Comment (eudoroolivares2016) [inline schemas/resources/schemas/collection-draft/v1.0.0/search.md:173]:
```suggestion
##### Valid Collection Draft Sort Keys
```

File: schemas/resources/schemas/collection-draft/v1.0.0/search.md [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+            </reference>
+        </references>
+    </results>
+```
+
+##### JSON
+
+The JSON response includes the following fields.
+
+* hits - How many total Order Options were found.
```
[the remark this answers is not in the document]
Comment (eereiter) [reply schemas/resources/schemas/collection-draft/v1.0.0/search.md:84]: I thought I found them all, looks like I missed some. 
Thanks for finding them.
I fixed it.

File: schemas/resources/schemas/collection-draft/v1.0.0/search.md (hunk 1 above)
Comment (eereiter) [reply schemas/resources/schemas/collection-draft/v1.0.0/search.md:86]: Fixed it.

**Review by sxu123 — APPROVED** [review]

File: schemas/resources/schemas/collection-draft/v1.0.0/search.md [hunk 4]
```diff
@@ -0,0 +1,184 @@
+### <a name="collection-draft"></a> Collection Draft
+
+Collection Drafts are draft records that describe a data set.Collection metadata is stored in the JSON format [UMM-Collection Schema](https://git.earthdata.nasa.gov/projects/EMFD/repos/unified-metadata-model/browse/collection).
```
Comment (eudoroolivares2016) [inline schemas/resources/schemas/collection-draft/v1.0.0/search.md:3]: nit;
```suggestion
Collection Drafts are draft records that describe a data set. Collection metadata is stored in the JSON format [UMM-Collection Schema](https://git.earthdata.nasa.gov/projects/EMFD/repos/unified-metadata-model/browse/collection).
```

File: schemas/resources/schemas/order-option-draft/v1.0.0/ingest.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+```
+    <?xml version="1.0" encoding="UTF-8"?>
+    <result>
+        <concept-id>OOD1200000000-PROV1</concept-id>
+        <revision-id>1</revision-id>
+        <warnings></warnings>
+        <existing-errors></existing-errors>
+    </result>
+```
+Subsequent ingests to the Oder Option Draft record will result in updates to it's metadata as well as increment the revision-id of the record.
```
Comment (eudoroolivares2016) [inline schemas/resources/schemas/order-option-draft/v1.0.0/ingest.md:29]:
```suggestion
Subsequent ingests to the Order Option Draft record will result in updates to it's metadata as well as increment the revision-id of the record.
```

**Review by eudoroolivares2016 — APPROVED** [review]

**Pull request merged**